src/api-reference/budget/budget-header.md

```diff
@@ -581,4 +581,34 @@ Name | Type | Format | Description
 `syncGuid`	|	`string`	|	-	|	The budget service's key for this object.
 `spendDate` |   `date`  |   -   |   If the current date is after this fiscal period's start date, this field shows the current date.  **READ ONLY**
 
+### Errors
+HTTP Error Code | Description
+---|---
+`200`|`OK - Successful call, response is in body.`
+`400`|`Bad Request - The request was determined to be invalid by the server. Possibly a validation failed on the data that was sent in the payload. For example, a Budget Item needs to have a name longer than 1 character. The response will have a list of validation errors in the body. See below for an example 400 response.`
+`401`|`Unauthorized - The user could not be authenticated.`
+`403`|`Forbidden - The user does not have the necessary permissions to perform the request`
+`404`|`Not Found - The resource could not be found or does not exist`
+`500`|`Server Error - Error message in response body`
+`503`|`Server Timeout - Error message in response body`
+
+### Example 400 Response with JSON
+
+**Response**
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+```
+```json
+{
+  "status" : false, 
```

Comment:
Something is wrong with either the markup or the JSON here.

Reply:
This should be fixed now, there was a missing " way above in the document

Reply:
Excellent.